### DISCUSSION TITLE
Append data by refresh

### DISCUSSION THREAD

**Discussion by maxdzin** [nuxt/nuxt#16239] [Ideas]
Hi Nuxters!

I just worked on infinite (scroll) pagination (for now, I managed that by using the watcher). And I thought that the built-in option to append the data by `refresh` (returned from `useFetch`, `useAsyncData`) might be helpful to have.

Maybe someone already has some clever solution, so it would be nice to know about it if you can share.

**Comment by mctweb** [comment]
https://vueuse.org/core/useinfinitescroll/#useinfinitescroll

**Reply by maxdzin** [reply]
@mctweb Thank you, I already tried useInfiniteScroll. There was a problem that the container must have a fixed height which is not always possible to achieve.
